md/project_deploy_in_studio.md

```diff
@@ -0,0 +1,36 @@
+# Project deployment in a Studio
+
+This page presents an overview of the _Deploy wizard_ capabilities in the Studio
+
+## Artifact selection
+
+After importing or cloning a project in the Studio, you will have the possibility to directly deploy your project in the embedded studio runtime.  
```

Comment:
either "Bonita Studio" or "the studio".
No "the Bonita Studio", 
No "the Studio"
No "Bonita studio"
#shrug :]